### DISCUSSION TITLE
AIO 8443/8080 MasterPasswort

### DISCUSSION THREAD

**Discussion by krauthaus** [nextcloud/all-in-one#657] [Questions] [answered]
Hi there! When I was away for three days our Nextcloud stopped working; unfortunately the master password for the backend seems to be not working. After a reboot I can login via ssh again; so I got access to the system over my domain, the local network and with keyboard mouse.

Is there any way to assign a new master Passwort or a place where I could find the actual one? 

I want to get to the point where I can access the logs. If someone could alternatively hint me to the right directories where I can find the logs I would be very thankful! (I got Backups via BackupApp and Borg)

THX

**Comment by szaimen** [comment, marked as answer]
Hello, 

you should be able to get the mastercontainer password by running
`sudo cat /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json | grep password`
(It is the value of `password`).

The logs of the mastercontainer should be visible by running `sudo docker logs nextcloud-aio-mastercontainer`.

**Reply by krauthaus** [reply]
Thank you very much for your fast and kind answer!
